### DISCUSSION TITLE
How to prevent ended job message from being cleared when using multiline prompt?

### DISCUSSION THREAD

**Discussion by lacygoill** [fish-shell/fish-shell#8742] [Q&A] [answered]
Consider this minimal prompt:
```fish
function fish_prompt
    printf '\n$ '
end
```
Written in `/tmp/home/.config/fish/config.fish`.

Start fish like this:

    $ XDG_CONFIG_HOME=/tmp/home/.config fish

Finally, start a job:

    $ sleep 1 &

When the job terminates, it writes a message on the terminal:

    $ fish: Job 1, 'sleep 1 &' has ended

But most of the time, this message is automatically cleared (*maybe* 5 out of 6 times):

![gif](https://user-images.githubusercontent.com/8505073/155185829-df63a3f2-180b-450c-80dd-bed0db0f0548.gif)

The issue disappears if the newline is removed from the prompt.

Is there a way to prevent the message from being cleared?

**Comment by zanchey** [comment, marked as answer]
Cute, I think this is a bug. #8817
